src/modules/SignIn/components/SignIn/components/FormSignIn/FormSignIn.jsx

```diff
@@ -46,35 +52,78 @@ export default class FormSignIn extends Component {
     onSubmit(event) {
         event.preventDefault();
         const { formData } = this.state;
-        const validateResult = validSignInForm(formData);
-
-        if (validateResult.status) {
-            API.signIn(formData)
-                .then((response) => {
-                    if (response.status === signInSuccessCode) {
-                        setTokens(response.data.access, response.data.refresh);
-                        window.location.href = "/dashboard";
-                    }
-                })
-                .catch((error) => {
-                    if (error.response.status === signInErrorCode) {
+
+        API.signIn(formData)
+            .then((response) => {
+                if (response.status === signInSuccessCode) {
```

Comment:
Let's not defined `signInSuccessCode`, `signInErrorCode`, `signUpSuccessCode` and `signUpErrorCode` in configs.